packages/backend/src/App.ts

```diff
@@ -396,6 +396,10 @@ export default class App {
                         name: errorResponse.name,
                         message: errorResponse.message,
                         data: errorResponse.data,
+                        id:
+                            errorResponse.statusCode === 500
+                                ? Sentry.lastEventId()
+                                : undefined,
```

Comment:
Code that returns error id.
```expressApp.use(Sentry.Handlers.errorHandler());```
The Sentry error handler already records the exception so we just need to grab the last event id.